website/docs/about/introduction/glossary.md

```diff
@@ -81,6 +81,9 @@ queues/etc to allow for processing of failed messages.
 For more on DLQs, see the [Amazon Documentation](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html)
 and the [Cumulus DLQ feature page](https://nasa.github.io/cumulus/docs/features/dead_letter_queues).
 
+### Deep Archive Storage
+[Amazon S3 Deep Archive](https://aws.amazon.com/s3/storage-classes/#____) is Comparable to [Amazon S3 Glacier](#glacier-storage) but with a lower cost in exchange for slower retrieval time.
```

Comment:
Wondering if we should change `Amazon S3 Glacier` to `Amazon S3 Glacier Flexible Retrieval` to better align with AWS name changes from last year. https://aws.amazon.com/s3/storage-classes/glacier/

Reply:
Wow, that name is awful. Updated down in the section on Glacier. Given that we still use the simple name in most places, that might be the extent of what we can do here.